src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -8,46 +8,52 @@
 # pylint: disable=line-too-long, too-many-lines
 helps['managedapp'] = """
     type: group
-    short-summary: Manage template solutions provided and maintained by the ISV using managedapp and managedapp definitions.
+    short-summary: Manage template solutions provided and maintained by Independent Software Vendors (ISVs).
 """
 helps['managedapp definition'] = """
     type: group
-    short-summary: Manage managed application definitions.
+    short-summary: Create, delete, and manage applications and their definitions.
```

Comment:
As awkward as the orginal wording was, the "thing" is called "managed application" so the second one isn't technically accurate. I'd keep the original.